_posts/2017-10-27-interview-with-patricia-falcao-francesca-colussi-tate.md

```diff
@@ -0,0 +1,60 @@
+
+MediaConch is particularly useful when I prepare files for multiple channel installations that originally have the same or similar properties. For example if we use the same setting to encode all the files and some of them appear faulty I use MediaConch to create a local policy and spot the issues. Sometimes it’s challenging as the rules of dependency between equipment and files vary and Dave Rice helped us exploring the possibility to develop a specific policy to check the files against the Brightsign specs which I find a very interesting challenge.  
+
+**At what point in the archival process do you use MediaConch?**  
+
+PF: We would usually use MediaConch either when we receive new files, before preparing an AIP/Bag for storage or after a migration from Tape to file.  
+
+**Do you use MediaConch for MKV/FFV1/LPCM video files, for other video files, for non-video files, or something else?**  
+
+PF: So far we’ve used it for Quicktime files with either V210, ProRes, DV or H264 compression, but also for other video files, like Mpeg.  
+
+FC: I mainly use it for video or sound files, ProRes, H264 and Mpeg are the most common files I handle.  
+
+**Why do you think file validation is important?**  
+
+PF: Because it will highlight any issues that a file may have that may not be immediately visible in the image or even the metadata but that may impact a file’s sustainability in the long-term.   
+
+FC: I totally agree with Patricia. The previous step in validation done during the acquisition process makes sure we have an issues free master file to refer to when producing any exhibition formats.  
+
+**Anything else you'd like to add?**
+
+PF: over the last few years, through the collaboration with Dave Rice and tool developers we have been completely sold to using open source tools for our archiving workflows. It is such an impressive community working in this field. Congratulations!  
+
+*Thanks so much! For more from Tate, see this whitepaper on [Audiovisual Adherence](http://www.tate.org.uk/research/publications/audiovisual-adherence), co-authored with our own Dave Rice. The Tate team also recently produced a short video entitled Things Change: Conservation and Display of Time-based Media Art, available [here](http://www.tate.org.uk/about-us/projects/pericles/things-change-conservation-and-display-time-based-media-art), that explains their conservation process and includes MediaConch usage!*
```

Comment:
Seems good. Would suggest to correct spelling to “QuickTime” and “QCTools”.

Reply:
Oh, thanks! I caught ONE of the QCTools, among other things, but missed the other two right next to it. Corrected it and QuickTime too.